environment/src/main/java/com/sequenceiq/environment/parameters/v1/converter/AzureEnvironmentParametersConverter.java

```diff
@@ -43,6 +44,10 @@ protected void postConvert(BaseParameters baseParameters, Environment environmen
                 .map(AzureParametersDto::getAzureResourceGroupDto)
                 .map(AzureResourceGroupDto::getResourceGroupUsagePattern)
                 .orElse(null));
+        azureParameters.setResourceEncryptionParameterKeyUrl(azureParametersDto
```

Comment:
Please add unit tests for the changed parts of `AzureEnvironmentParametersConverter`.